apps/ima/views/content/life-or-liberty-threatened.md

```diff
@@ -0,0 +1,10 @@
+<span class="threatened-paragraph">Your life of liberty must have been threatened because of your:</span>
```

Comment:
what is the purpose of this class? There doesn't seem to be any styling for it in the css file. If it is not needed you can remove the span tag completely.